src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected double area;
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public double getArea() {
+        return area;
+    }
+
+    @Override
+    public void draw() {
+    }

```

Comment:
no need to implement this here. 
`getArea` should calculate area by side lengths, not just to return some `area` field (which should be deleted)
```suggestion
```